shared/src/commonMain/kotlin/ui/views/CredentialDetailsView.kt

```diff
@@ -90,18 +97,27 @@ fun CredentialDetailsScaffold(
                     NavigateUpButton(navigateUp)
                 },
                 title = {
-                    Text(
-                        stringResource(Res.string.heading_label_credential_details_screen),
-                        style = MaterialTheme.typography.headlineLarge
-                    )
+                    Row(Modifier.fillMaxWidth(), verticalAlignment = Alignment.CenterVertically) {
+                        Text(
+                            stringResource(Res.string.heading_label_credential_details_screen),
+                            modifier = Modifier.weight(1f),
+                            style = MaterialTheme.typography.headlineLarge
+                        )
+                        Image(
+                            modifier = Modifier.padding(start = 0.dp, end = 0.dp, top = 8.dp),
+                            painter = painterResource(Res.drawable.asp),
+                            contentDescription = null,
+                            contentScale = ContentScale.Fit,
+                        )
+                    }
                 },
-                actions = {
+                /*actions = {
```

Comment:
Why is this code commented-out?

Reply:
removes the delete button from the details view, because it does not work with the logo at all. There's simply no way to have the logo and the three-dots menu here is a satisfying way.

Reply:
So then w'ell need to add a button to the details view and remove the code here?

Reply:
the button is already in the credentials list, so you can still remove it. where this button would fit in this view then, I don't know.

Reply:
brought the menu back (see screenshots below)